docs/HowTo/Quickstart.md

```diff
@@ -1,31 +1,31 @@
 # Getting Started
 
-## Prequisites
+## Prerequisites
 
 [Orion](../HowTo/Install-Binaries.md)
 
 ## Starting Orion and Sending a Payload
 
-### 1. Create Password File
+### Create Password File
```

Comment:
Create password file 

Needs to be sentence case not title case. Applies to following sections also.

Reply:
This is being fixed by https://github.com/PegaSysEng/doc.orion/pull/20